app/invocation/invocation.css

```diff
@@ -55,6 +55,10 @@
   background-color: #eee;
 }
 
+.content .button {
+  float: right;
```

Comment:
we generally try to avoid float and use flexbox instead, floating elements cause a bunch of problems: https://medium.com/dvt-engineering/css-flexbox-a-useful-alternative-to-div-and-float-551c98d26aeb